### PR TITLE
Adds More Achievements

### PR DESCRIPTION
**Pull request by Ivniinvi**
## About The Pull Request
Adds more achievements:
Veteran: 1000 Living Hours
Seasoned Veteran: 2000 Living Hours
Grilled Seasoned Veteran: 3000 Living Hours
Sous Vide Grilled Seasoned Veteran: 4000 Living Hours
Best Meal of My Life: Get gibbed by Mint Toxin
Mind the Gap: Get saved from a chasm by a jaunter
This Lousy Achievement: Be part of a cargo crew that buys a bike
Let's Play GLOBAL THERMONUCLEAR WAR: Be part of a nuke ops team that declares war
Little Chickadee: Have a drunkenness of 250 or greater

## Why It's Good For The Game
These are all things that are funny, hard, or deserve to be commemorated, in my opinion.

## Testing Photographs and Procedure

<details>

<summary>Screenshots&Videos</summary>

Put screenshots and Videos documenting testing and execution of intended behaviors here

</details>

## Changelog
:cl:
add: Added 9 new achievements!
/:cl:

### CODE REVIEW

**Comment by MCterra10** [comment]
I'd be surprised if even a single player has more than 4000 hours

**Comment by Ivniinvi** [comment]
> I'd be surprised if even a single player has more than 4000 hours

We currently have one player with over 3000, to my knowledge, and when someone hits 4000 I'll add a 5000 achievement, and so on and so forth

**Comment by Ivniinvi** [comment]
Ignore the mistakes with git

File: code/__HELPERS/roundend.dm [hunk 1]
```diff
@@ -269,6 +269,20 @@
 
 	CHECK_TICK
 
+	//Process veteran achievements
+	for(var/client/C in GLOB.clients)
```
Comment (Archanial) [inline code/__HELPERS/roundend.dm]: As()

File: code/modules/antagonists/nukeop/equipment/nuclear_challenge.dm [hunk 2]
```diff
@@ -66,6 +66,7 @@
 	for (var/datum/mind/M in get_antag_minds(/datum/antagonist/nukeop))
 		if (iscyborg(M.current))
 			continue
+		M.current.client.give_award(/datum/award/achievement/misc/warops, M.current)
```
Comment (Archanial) [inline code/modules/antagonists/nukeop/equipment/nuclear_challenge.dm]: Don't assume client

File: code/datums/achievements/misc_achievements.dm [hunk 3]
```diff
@@ -92,3 +92,47 @@
...
+
+/datum/award/achievement/misc/onekhours
+	name = "Veteran"
+	desc = "You've spent a lot of time on this game, have a reward!"
+	database_id = MEDAL_ONEKHRS
+	reward = 1000
+
+/datum/award/achievement/misc/twokhours
+	name = "Seasoned Veteran"
+	desc = "Consider applying for Mentor"
```
Comment (Archanial) [inline code/datums/achievements/misc_achievements.dm]: Don't like this desc

File: code/modules/antagonists/nukeop/equipment/nuclear_challenge.dm (hunk 2 above)
Comment (Archanial) [reply code/modules/antagonists/nukeop/equipment/nuclear_challenge.dm]: Here and in the give award procs

File: code/modules/shuttle/supply.dm [hunk 4]
```diff
@@ -149,6 +149,10 @@ GLOBAL_LIST_INIT(blacklisted_cargo_types, typecacheof(list(
 		investigate_log("Order #[SO.id] ([SO.pack.name], placed by [key_name(SO.orderer_ckey)]), paid by [D.account_holder] has shipped.", INVESTIGATE_CARGO)
 		if(SO.pack.dangerous)
 			message_admins("\A [SO.pack.name] ordered by [ADMIN_LOOKUPFLW(SO.orderer_ckey)], paid by [D.account_holder] has shipped.")
+		if(istype(SO, /datum/supply_pack/misc/bicycle))
```
Comment (Archanial) [inline code/modules/shuttle/supply.dm]: This should be handled in the pack itself. Doesn't it have some on_buy proc?

**Review by Archanial — DISMISSED** [review]

File: code/__HELPERS/roundend.dm (hunk 1 above)
Comment (ike709) [reply code/__HELPERS/roundend.dm]: Always be prepared for the client to be impossibly null.

**Comment by Crossedfall** [comment]
> I'd be surprised if even a single player has more than 4000 hours

Current top is 3959 hours.

**Review by ike709 — APPROVED** [review]

**Review by Crossedfall — APPROVED** [review]

**Pull request merged**